app/pages/ServiceDiscoveryForm/ServiceDiscoveryForm.tsx

```diff
@@ -114,25 +165,48 @@ const Content = ({
     );
   };
 
+  const handleRadioSelect = (targetCategoryId: number) => {
+    setSelectedSubcategories({ [targetCategoryId]: true });
+    setSelectedRadioItem(targetCategoryId);
+  };
+
   switch (steps[currentStep]) {
+    case 'housingStatus':
+      return (
+        <RadioFormStep
+          heading="Which of the following best describes your current housing situation?"
+          subheading="Select the option most relevant to you."
+          handleRadioSelect={handleRadioSelect}
+          refinements={subcategories}
+        />
+      );
     case 'eligibilities':
       return (
         <FormStep
           heading="Tell us more about you"
           subheading="Select any eligibilities you identify with."
-          options={eligibilities}
-          selectedOptions={selectedEligibilities}
-          toggleOption={handleEligibilityClick}
+          refinements={eligibilities}
+          selectedRefinements={selectedEligibilities}
+          toggleRefinement={handleEligibilityClick}
         />
       );
     case 'subcategories':
       return (
         <FormStep
           heading="Tell us more about you"
           subheading={subcategorySubheading}
-          options={subcategories}
-          selectedOptions={selectedSubcategories}
-          toggleOption={handleSubcategoryClick}
+          refinements={subcategories}
+          selectedRefinements={selectedSubcategories}
+          toggleRefinement={handleSubcategoryClick}
+        />
+      );
+    case 'subcategoriesRadio':
+      return (
+        <RadioFormStep
```

Comment:
I think at least a few of our current pathways should be using a radio form instead of checkboxes as we only want users to select on refinement at a time in some cases.